### PR TITLE
Game/cc 26

### PR DESCRIPTION
**Pull request by skeola**
Added a screen after end selection to manage current game status.

### CODE REVIEW

File: components/PreGameScreen/PreGameScreen.js [hunk 1]
```diff
@@ -6,7 +6,7 @@ import {
   removeOrientationListener
 } from "react-native-responsive-screen";
 import Settings from "../Settings/Settings";
-import StartGame from "../StartGame/StartGame";
+// import StartGame from "../StartGame/StartGame";
```
Comment (hendersonreed) [inline components/PreGameScreen/PreGameScreen.js:9]: Would be great if this could just be deleted, since it's not necessary anymore.

**Comment by hendersonreed** [comment]
Initial test on Android works for me, this looks great! I'll leave it up to Harriet for final review, since she's the one who knew what the story was supposed to do, but this looks like it passes an initial glance-over.

**Comment by hpadkins** [comment]
@hendersonreed Uh oh, is it not clear what the story is about?

File: components/StartGame/StartGame.js [hunk 2]
```diff
@@ -4,14 +4,40 @@ import { View, Text, Image} from 'react-native';
...
+    };
+  }
+
+  displayCurrentPlayer(curentShot) {
+    if(curentShot < 2) {
+      return <Text style={styles.player}>PLAYER 1:</Text>
+    } else if (curentShot < 4) {
+      return <Text style={styles.player}>PLAYER 2:</Text>
+    } else if (curentShot < 6){
+      return <Text style={styles.player}>PLAYER 3:</Text>
```
Comment (hpadkins) [inline components/StartGame/StartGame.js:21]: These should be `currentShot`, no?

Comment (hpadkins) [reply components/StartGame/StartGame.js:21]: This is likely causing the error where the game starts with Player 4

**Comment by hendersonreed** [comment]
We still need the merge conflicts fixed here before we can merge, as well as any other concerns Harriet raised. The sooner we can do this though, the sooner we can shut down this repo and move over to gitlab entirely.

**Comment by hendersonreed** [comment]
Merging this now so we can get going in the new repo on Gitlab - https://gitlab.com/curling-companion/curling-companion

**Pull request merged**